contributing/index.md

```diff
@@ -7,14 +7,33 @@ has_children: false
 
 # Contribute to Data Commons!
 
-Data Commons has benefited greatly from our collaborations with different government organizations and academic institutions and are looking to expand the set of collaborative projects. In particular, we are looking for partners to:
+Data Commons has benefited greatly from our collaborations with different government organizations and academic institutions and is looking to expand the set of collaborative projects. In particular, we are looking for partners to:
 
+- [Add data](#add-data): Contribute data to the graph which will be accessible via Data Commons tools and APIs.
 - [Create tools](#creating-a-new-tool): Build new tools or applications that bring the data in Data Commons to new categories of users.
 - [Create new curriculum](#sharing-analysis): Using Data Commons in data science and machine learning courses.
 - [Write documentation](#updating-documentation)
 
 Data Commons welcomes patches and contributions to this project from everyone, new or experienced!
 
+## Ways to contribute
+
+### Add data
+
+We welcome contributions of public data to the Data Commons Graph. We’ve bootstrapped the graph with [data](/datasets) from US Census, World Bank, CDC, NOAA, IPCC, etc. However, Data Commons is meant to be for the community, by the community. We're excited to work with you to make public data accessible to everyone. Please see [the Data Commons data repository](https://github.com/datacommonsorg/data) for more information, and a [list of imports](https://github.com/orgs/datacommonsorg/projects/10/views/1) that would be helpful to add.
```

Comment:
It feels useful to link to both the top-level doc (for dev process) and https://github.com/datacommonsorg/data/tree/master/docs (for concepts)?

*To get started, please take a look at the resources listed [here](https://github.com/datacommonsorg/data/tree/master/docs#contributing-to-data-commons) and follow the development steps outlined [here](https://github.com/datacommonsorg/data#development).*

or something like that..

Reply:
ah, good suggestion! done